doc/whats_new.rst

```diff
@@ -228,6 +229,10 @@ API changes summary
      faster than before and some of computationnal problems have been solved.
      By `Wei Xue`_ and `Thierry Guillemot`_.
 
+   - The ``grid_scores_`` attribute of :class:`model_selection.GridSearchCV` and
+     :class:`model_selection.RandomizedSearchCV` is deprecated in favor of the
+     attribute, ``results_``, which is a dict of 1D (numpy) (masked) arrays. The per-split score, mean, std and rank for each parameter setting (search candidate) are stored as separate numpy arrays under corresponding keys in the ``results_`` dict.  The parameter values for each parameter is stored separately as a numpy masked object array. The value, for that search candidate, is masked if the corresponding parameter is not applicable. Additionally a list of all the parameter dicts are stored at ``results_['parameters']``. This ``results_`` dict can be easily imported into pandas as a dataframe for exploring the search results. By `Raghav RV`_.
+
```

Comment:
> However, I think the new model selection changes really need a section of their own in the 0.18 what's new, as it touches a lot of people's code; 

Do you mean moving all the entries to that section or ...
keeping the entries in the respective sections brief and referring to the `Changes to the model selection` section, where we explain it in more detail?